project_report/project_report_latex.py

```diff
@@ -160,6 +116,132 @@ def create_horizontal_table(container, rows):
+            NoEscape(r'Library prep.\textsuperscript{2}'),
+            NoEscape(r'Analysis\textsuperscript{3}')
+        ]
+        # Latex specific column definition
+        # Set the column width to fix width for all but first column
+        column_def = 'X[l] p{2.5cm} p{1.7cm} p{1.7cm} p{1.5cm} p{1.8cm} p{1.7cm}'
+
+        def find_sample_release_date_in_auth(sample):
+            return [auth.get('date') for auth in authorisations if sample in auth.get('samples')]
+
+        rows = []
+
+        library_descriptions = set()
+        species_descriptions = set()
+        analysis_descriptions = set()
+        for sample in self.pi.sample_names_delivered:
+            date_reviewed = find_sample_release_date_in_auth(sample)
+            internal_sample_name = self.pi.get_fluidx_barcode(sample) or sample
+            library = self.pi.get_library_workflow_from_sample(sample)
+            library_descriptions.add((self.pi.library_abbreviation.get(library), library))
+            species = self.pi.get_species_from_sample(sample)
+            species_descriptions.add((self.pi.abbreviate_species(species), species))
+            analysis = self.pi.get_analysis_type_from_sample(sample)
+            analysis_descriptions.add((self.pi.analysis_abbreviation.get(analysis),
+                                       self.pi.analysis_description.get(analysis)))
+            row = [
+                self.pi.get_user_sample_id(sample),
+                internal_sample_name,
+                parse_date(self.pi.get_started_date_from_sample(sample)),
+                ', '.join(date_reviewed),
+                self.pi.abbreviate_species(species),
+                self.pi.library_abbreviation.get(library),
+                self.pi.analysis_abbreviation.get(analysis)
+            ]
+
+            rows.append(row)
+        tables['appendix I'] = {
+            'header': header, 'column_def': column_def, 'rows': rows,
+            'footer': [
+                self.format_table_footer_line(sorted(library_descriptions), 1),
+                self.format_table_footer_line(sorted(species_descriptions), 2),
+                self.format_table_footer_line(sorted(analysis_descriptions), 3)
+            ]
+        }
+        header = [
+            'User ID', 'Yield quoted (Gb)', 'Yield provided (Gb)', '% Q30 > 75%', 'Quoted coverage', 'Provided coverage'
+        ]
+
+        rows = []
+        for sample in self.pi.sample_names_delivered:
+            row = [
+                self.pi.get_user_sample_id(sample),
+                self.pi.get_required_yield(sample),
+                self.pi.get_yield_in_gb(sample),
+                self.pi.get_pc_q30(sample),
+                self.pi.get_quoted_coverage(sample),
+                self.pi.get_average_coverage(sample)
+            ]
+
+            rows.append(row)
+        tables['appendix II'] = {'header': header, 'rows': rows}
+
+        return tables
```

Comment:
`get_project_info`, `format_result_summary`, `format_table_footer_line` and `get_sample_data_in_tables` are functions that were in `ProjectReportInformation` before and have been move in `ProjectReportLatex` because they perform some data formatting.